lib/minimal.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+module.exports.register = function ({ playbook, config }) {
+    const logger = this.getLogger('andora')
```

Comment:
There's no need to pass a value here. "antora" (if spelled correctly) is already the default category.